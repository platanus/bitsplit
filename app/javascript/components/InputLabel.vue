<template>
  <label
    class="block text-gray-700 text-sm font-bold mb-2"
    for="fieldFor"
  >
    <slot />
  </label>
</template>
<script>
export default {
  name: 'InputLabel',
  props: {
    fieldName: String,
    fieldFor: String,
  },
};
</script>
