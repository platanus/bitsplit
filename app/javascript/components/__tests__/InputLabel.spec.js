import { shallowMount } from '@vue/test-utils'
import InputLabel from '../InputLabel'

describe('InputLabel', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(InputLabel, {
      propsData: {
        fieldName: 'name',
        fieldFor: 'for'
      }
    })
    expect(wrapper).toMatchSnapshot()
  })
})
