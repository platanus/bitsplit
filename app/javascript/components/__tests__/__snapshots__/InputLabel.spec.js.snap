// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputLabel matches snapshot 1`] = `<label for="fieldFor" class="txt-field text-left block text-gray-700 text-sm font-bold mb-2"></label>`;
